app/views/components/SaveButtonsV2.scala.html

```diff
@@ -0,0 +1,32 @@
+@*
```

Comment:
Can be rename this to something more explicit?

maybe SaveButtonsWithToDraft?

Reply:
I like that suggestion 👍